Gruntfile.js

```diff
@@ -157,26 +161,39 @@ grunt.registerTask( "build-demos", function() {
 		grunt.file.copy( filename, targetDir + "/external/" + filename.replace( externalDir, "" ) );
 	});
 
-	function replaceResources( source ) {
-		// ../../jquery-x.y.z.js -> CDN
-		source = source.replace(
-			/<script src="\.\.\/\.\.\/external\/jquery\/jquery\.js">/,
-			"<script src=\"//code.jquery.com/jquery-" + jqueryCore + ".js\">" );
+	function deAmd( $, destDir ) {
+		var bootstrap = $( "script[src='../bootstrap.js']" ),
+			require = $( "script[src='../../external/requirejs/require.js']" ),
+			i18n = bootstrap.attr( "data-modules" );
+
+		// Replace the src on the first tag with core
+		require.attr( "src", "https://code.jquery.com/jquery-" + jqueryCore + ".js" )
+
+			// Add a script tag for UI after
+			.after( "\n\t<script src=\"https://code.jquery.com/ui/" +
+				stable.pkg.version + "/jquery-ui.js\"></script>" );
+
+		// Any modules that are not part of the actual library must contain .js and are added as
+		// new script tags before the bootstrap file
+		if ( i18n ) {
+			i18n = i18n.match( /\S+/g ).filter( function( value ) {
+				return /i18n/.test( value );
+			} );
+			if ( i18n.length ) {
+				for( var i = 0; i < i18n.length; i++ ) {
+					grunt.file.copy( repoDir + "ui/" + i18n[ i ] + ".js", destDir + i18n[ i ] );
+					bootstrap.before( "<script src=\"" + i18n[ i ] + ".js\"></script>\n\t" );
+				}
+			}
+		}
 
-		// ../../ui/* -> CDN
-		// Only the first script is replaced, all subsequent scripts are dropped,
-		// including the full line
-		source = source.replace(
-			/<script src="\.\.\/\.\.\/ui\/[^>]+>/,
-			"<script src=\"//code.jquery.com/ui/" + stable.pkg.version + "/jquery-ui.js\">" );
-		source = source.replace(
-			/^.*<script src="\.\.\/\.\.\/ui\/[^>]+><\/script>\n/gm,
-			"" );
+		bootstrap
+			.removeAttr( "src" )
+			.html( "\n\t$( function() {" + bootstrap.html() + "} );\n\t" )
+			.removeAttr( "data-modules" );
```

Comment:
Rather than modifying the two existing scripts, I'd rather generate new elements and remove the existing ones. It's less fragile.